ecosystem/python/sdk/aptos_sdk/client.py

```diff
@@ -459,6 +466,7 @@ def get_token_balance(
             token_id,
         )["amount"]
 
+        #:!:>read_token_data_table
```

Comment:
This should be at the edge rather than aligned?  there seems to be an inconsistency here

Reply:
will re-run the autoformatters just in case